djangoplicity/reports/models.py

```diff
@@ -6,10 +6,14 @@
 #   Luis Clara Gomes <lcgomes@eso.org>
 #
 from django.db import models
-from django.core.urlresolvers import reverse
 from django.utils.translation import ugettext_lazy as _
 from django.utils.encoding import python_2_unicode_compatible
 
+import django
+if django.VERSION >= (2, 2):
```

Comment:
Same comment as before, `django.core.urlresolvers` was removed in django 2.0.

Reply:
@danialeresbar Please update this.